api/src/telemetry/lib/send-owner-report.ts

```diff
@@ -0,0 +1,30 @@
+import { useEnv } from '@directus/env';
+import type { OwnerInformation } from '@directus/types';
+import { URL } from 'node:url';
+
+/**
+ * Post owner report to the centralized intake server
+ */
+export async function sendOwnerReport(report: OwnerInformation & { project_id: string }) {
+	const env = useEnv();
+
+	const url = new URL('/v1/owner', env['TELEMETRY_URL'] as string);
```

Comment:
Can this new datapoint be added into the existing telemetry fn, rather than needing a new function here?